webdriver/webdriver.go

```diff
@@ -30,15 +30,41 @@ func GetWebDriver() (*selenium.Service, selenium.WebDriver, error) {
 	panic("Invalid --browser value specified")
 }
 
-func FindShadowElements(d selenium.WebDriver, e selenium.WebElement, selector string) ([]selenium.WebElement, error) {
-	i := []interface{}{e}
-	result, err := d.ExecuteScriptRaw(fmt.Sprintf("return arguments[0].shadowRoot.querySelectorAll('%s')", selector), i)
-	if err != nil {
-		panic(err.Error())
+func FindShadowElements(
```

Comment:
I saw [your comment in the other PR](https://github.com/web-platform-tests/wpt.fyi/pull/384/files#r205461237). This is quite useful, but can we add a short comment to explain the arguments (especially whether the behaviour is similar to CSS selector `a b c` or `a > b > c`; if I read the code correctly, it's the former)?